API.md

```diff
@@ -0,0 +1,41 @@
+# SegM8 API
+
+## `class SegM8`
+
+Creates an object of type `SegM8` to communicate with a chain of particular [SegM8 7-segment indicator modules](https://my.amperka.com/modules/SegM8).
+
+### `SegM8(uint8_t pinCS, uint8_t devicesCount = 1)`
+
+Constructs a new SegM8 object that uses the default hardware SPI bus.
+
+- `pinCS`: the chip select (also known as slave select) pin used to control the shift-register latch. It is marked `CS` on the SegM8 board. Any GPIO pin of the controller may be used.
+- `deviceCount`: the number of SegM8 modules connected in a daisy-chain. If omitted, defines a single module.
+
+### `SegM8(uint8_t pinCS, uint8_t pinMOSI, uint8_t pinSCK, uint8_t devicesCount = 1)`
+
+Constructs a new object of data bus Software SPI with arguments:
```

Comment:
```suggestion
Constructs a new SegM8 object that uses a software SPI bus.
```

Reply:
Ditto as the description for Hardware SPI

Reply:
Ok